contrib/opentimelineio_contrib/adapters/ale.py

```diff
@@ -134,12 +134,30 @@ def _parse_data_line(line, columns, fps):
 
             if cdl:
                 del metadata['ASC_SOP']
+                asc_sop = cdl.get('asc_sop')
+                if asc_sop:
+                    line += "\t*ASC_SOP ({} {} {}) ({} {} {}) ({} {} {})\t".format(
+                        asc_sop['slope'][0],
+                        asc_sop['slope'][1],
+                        asc_sop['slope'][2],
+                        asc_sop['offset'][0],
+                        asc_sop['offset'][1],
+                        asc_sop['offset'][2],
+                        asc_sop['power'][0],
+                        asc_sop['power'][1],
+                        asc_sop['power'][2]
+                    )
```

Comment:
This looks like it's using EDL syntax to try and write to the ALE, but in the parsing code rather than the writer code?